Specs/DateTime/English/DateTimeModel.json

```diff
@@ -5285,5 +5285,30 @@
         }
       } 
     ]
+  },
+  {
+    "Input": "Show me sales in the year of 2008",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript, python",
+    "Results": [
+      {
+        "Text": "2008",
+        "Start": 29,
+        "End": 32,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2008",
+              "type": "daterange",
+              "start": "2008-01-01",
+              "end": "2009-01-01"
+            }
+          ]
+        }
+      }
+    ]

```

Comment:
if these test data are the same, maybe you can put them in another new common json file.